py/server/tests/test_table.py

```diff
@@ -902,6 +903,34 @@ def make_pairs_3(tid, a, b):
             self.assertEqual(x2.size, 10)
             self.assertEqual(x3.size, 10)
 
+    def test_class_attrs_in_query(self):
+        input_cols = [
+            datetime_col(name="DTCol", data=[dtypes.DateTime(1), dtypes.DateTime(10000000)]),
+        ]
+        test_table = new_table(cols=input_cols)
+        from deephaven.time import year, TimeZone
+        rt = test_table.update("Year = (int)year(DTCol, TimeZone.NY)")
```

Comment:
Maybe add a test case where you cast an argument in addition to the result of the expression.